### ISSUE TITLE
admin/content | remove the paging function

### ISSUE THREAD

**Issue by G3N-es** [bludit/bludit#1024]
### Describe your problem

Paging does not add anything, since bludit is able to manage hundreds of entries without slowing down.

Instead of helping, it creates difficulties.

It is much easier to drag down than to go to the next page.

THE MOST IMPORTANT!!!
Currently we do not have a search engine, but we can use the browser's search engine to locate articles.

If the list is divided into pages, or with the browser search can be located.

I had planned to develop a simple plugin for searches, but paging also complicates this.

To implement it would be as simple as the value for ITEMS_PER_PAGE_ADMIN was -1, and if you want to keep the paging functionality (although I insist I do not see sense) was variable.
https://github.com/bludit/bludit/blob/master/bl-kernel/boot/variables.php#L29

Another suggestion in this aspect is that the tags and categories appear in the list. This facilitates the location and search of articles.

---

La paginación no aporta nada, ya que bludit es capaz de gestionar cientos de entradas sin relentizarse.

En vez de ayudar, crea dificultades.

Es mucho más sencillo arrastrar hacia abajo que ir a la siguiente página.

LO MÁS IMPORTANTE!!!
Actualmente no disponemos de buscador, pero podemos echar mano del buscador del navegador para localizar artículos.

Si el listado está fragmentado en páginas, ni con el buscador del navegador pueden localizarse.

Tenía pensado desarrollar un simple plugin para búsquedas, pero la paginación también complica esto.

Para implementar sería tan sencillo como que el valor para ITEMS_PER_PAGE_ADMIN fuese -1, y si se quiere conservar la funcionalidad de paginación (aunque insisto no le veo sentido) fuese variable.
https://github.com/bludit/bludit/blob/master/bl-kernel/boot/variables.php#L29

Otra sugerencia en este aspecto es que aparezcan las etiquetas y categorías en el listado. Eso facilita la localización y búsqueda de artículos.

**Comment by dignajar** [comment]
Hi,
I implemented a search function in the content in Bludit v3.9.0 beta1. Take a look.
Regards

**Comment by G3N-es** [comment]
Muchas gracias Diego :-).
Mañana lo testeo en profundidad y te mando feed back.
Creo que era algo necesario, muchos lo estábamos demandando de una u otra forma (que si filtros que si esto que si lo otro) pero un buscador era lo más sencillo.

De veras. Gracias.

**Comment by G3N-es** [comment]
I think that searches should keep a similar logic in all sections of the admin.
Would not it be more successful to work as in plugins?
For example, the search could be placed to the right of the section title.
I insist on the need to show the categories and tags in the content list. It is essential.
In part I created G3Nshop to show you the need for this functionality.

Maybe his thing is to study it thoroughly and do it for version 4 of bludit.
Having a search engine in the admin sections makes a lot of sense, and you have to see what is the best way to implement it.

By the way there are bugs in the plugins search engine.
Try searching for TinyMCE

**Comment by BlakesHeaven** [comment]
The search in 'Content' admin is very useful,  and I agree on small things first, more in later versions. 
For me the priority order would be:

1. content to be sorted **decending** in admin so most recent are presented 'First' and the oldest 'Last'.

2. Add configurable setting for pagination, i.e. posts per screen, 10, 20 30, All

3. Version 4, it would be better still if you could select what/where you wanted to search via radio buttons, eg Title, Content, Category, Tag, ALL... and included more detail in search results, eg Date, Category, Tag.

**Comment by dignajar** [comment]
I fixed the plugin search.

I have bigger plans for the search tool, but for the next version, At this moment I want to release the new version and delivery some improves and new features.

Feel free to create new issues with the requests, but try to split in smaller topics, if I need to go to some ticket about `pagination` and I found a lot of recommendation or bugs about other things is hard for me to follow them.
